test/hotspot/jtreg/runtime/CompressedOops/CompressedClassSpaceSize.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 8022865
  * @summary Tests for the -XX:CompressedClassSpaceSize command line option
- * @requires vm.bits == 64 & vm.opt.final.UseCompressedOops == true
+ * @requires vm.bits == 64 & vm.opt.final.UseCompressedOops == true & vm.cds
```

Comment:
All test cases unrelated to CDS will also be disabled if the JVM is not configured with CDS.

I think it's better to move the new test cases to a new file.